File: 27/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

var scanner *bufio.Scanner

func init() {
	scanner = bufio.NewScanner(os.Stdin)
}

func eInput(msg string) string {
	fmt.Printf(msg)
	scanner.Scan()
	if scanner.Err() != nil {
		os.Exit(1)
	}
	return scanner.Text()
}

func main() {
	firstName := eInput("Enter the first name: ")
	lastName := eInput("Enter the last name: ")
	zipCode := eInput("Enter the ZIP code: ")
	employeeID := eInput("Enter an employee ID: ")
	ok, err := validateInput(firstName, lastName, zipCode, employeeID)
	if ok {
		fmt.Printf("There were no errors found.\n")
	} else {
		fmt.Printf(err.Error())
	}
}

File: 27/python/main.py
#!/usr/bin/env python
# coding: -utf8

if __name__ == '__main__':
    import validator as v

    first_name = input("Enter the first name: ")
    last_name = input("Enter the last name: ")
    zip_code = input("Enter the ZIP code: ")
    employee_id = input("Enter an employee ID: ")
    success, err = v.validate_input(
        first_name, last_name, zip_code, employee_id)
    if success:
        print("There were no errors found.\n", end='')
    else:
        print(err, end='')
